avocado/core/loader.py

```diff
@@ -122,6 +122,8 @@ def __init__(self):
         self._initialized_plugins = []
         self.registered_plugins = []
         self.reference_plugin_mapping = {}
+        self._label_mapping = None
+        self._decorator_mapping = None
```

Comment:
You're filling them with None and testing if they are not None. When do you fill them with something else?